elements/lisk-chain/src/utils/genesis_block.ts

```diff
@@ -14,8 +14,14 @@
 
 import { Schema, codec } from '@liskhq/lisk-codec';
 import { objects } from '@liskhq/lisk-utils';
+import { hash } from '@liskhq/lisk-cryptography';
 import { GenesisBlock, AccountSchema } from '../types';
-import { baseAccountSchema, getGenesisBlockHeaderAssetSchema, blockSchema, blockHeaderSchema } from '../schema';
+import {
+	baseAccountSchema,
+	getGenesisBlockHeaderAssetSchema,
+	blockSchema,
+	blockHeaderSchema,
+} from '../schema';
 
 export const readGenesisBlockJSON = (
```

Comment:
This function becomes too complex, we should refactor it later and build some easier pattern to mixin schemas together which is more readable.